azurerm/resource_arm_firewall_network_rule_collection.go

```diff
@@ -93,7 +93,6 @@ func resourceArmFirewallNetworkRuleCollection() *schema.Resource {
 								Type: schema.TypeString,
 								ValidateFunc: validation.StringInSlice([]string{
 									string(network.Any),
-									string(network.ICMP),
```

Comment:
Does this constant no longer exist or has it been renamed? Removing this value is a breaking change

Reply:
Same question, why removing it? The constant is still there: https://github.com/Azure/azure-sdk-for-go/blob/master/services/network/mgmt/2018-08-01/network/models.go#L423.

Reply:
Good catch. `ICMP` is not included in the v21.3 but included since v22.0. I will still keep the raw literal string here until we upgrade to v22.0.

Reply:
It's removed in v21.3 and added back in v22.0.